tests/virt_autotest/Prj1_Guest_Installation/guest_installation_run.pm

```diff
@@ -0,0 +1,64 @@
+# SUSE's openQA tests
+#
+# Copyright © 2009-2013 Bernhard M. Wiedemann
+# Copyright © 2012-2016 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+#
+use lib "/var/lib/openqa/share/tests/sle-12-SP2/tests/virt_autotest/lib";
```

Comment:
I agree with coolo, please put your libraries in the lib directory and just 'use' them